lnwallet/chainfee/estimator.go

```diff
@@ -164,34 +164,36 @@ func (b *BtcdEstimator) Start() error {
 		return err
 	}
 
-	// Once the connection to the backend node has been established, we'll
-	// query it for its minimum relay fee.
-	info, err := b.btcdConn.GetInfo()
+	// Once the connection to the backend node has been established, we
+	// can initialise the minimum relay fee manager which queries the
+	// chain backend for the minimum relay fee on construction.
+	minRelayFeeManager, err := newMinFeeManager(
+		defaultUpdateInterval, b.fetchMinRelayFee,
+	)
 	if err != nil {
 		return err
 	}
+	b.minFeeManager = minRelayFeeManager
+
+	return nil
+}
+
+// fetchMinRelayFee fetches and returns the minimum relay fee in sat/kb from
+// the btcd backend.
+func (b *BtcdEstimator) fetchMinRelayFee() (SatPerKWeight, error) {
+	info, err := b.btcdConn.GetInfo()
```

Comment:
@Roasbeef so this doesn't float according to mempool conditions? If that's true, then perhaps this doesn't need to be refreshed as it's effectively static?

Reply:
Correct, but no harm leaving it in IMO, since it makes the logic a bit more uniform.